<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_new_capture_source</class>
 <widget class="QFrame" name="f_new_capture_source">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Capture Source</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="cmb_capture_source_group"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_external_source_id"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l_external_source_id">
     <property name="text">
      <string>External Source ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_capture_source_group">
     <property name="text">
      <string>Capture Source Group:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btn_ok">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
